forms-flow-web/src/formComponents/RemoteSelect/index.js

```diff
@@ -0,0 +1,303 @@
+import { Components, Formio } from 'react-formio';
+import _ from 'lodash';
+import NativePromise from 'native-promise-only';
+import { WEB_BASE_URL } from "../../apiManager/endpoints/config";
+import UserService from "../../services/UserService";
+
+const SelectComponent = Components.components.select;
+
+export default class RemoteSelect extends SelectComponent {
```

Comment:
I'm wondering if we could add a "MinistrySelect" component that extends this that defaults the url to the ministry name endpoint, just to make it as easy to use as possible for a forms designer

Reply:
Sure that works. I'll look into it, amongst other ways to make it easier to use for a designer.

Reply:
Addressed. A new "MinistrySelect" was added that gets values from the `/ministry-names` endpoint by default. I also made some adjustments to `RemoteSelect` to only support URL-based values by default.